packages/link-modules/src/loaders/container.ts

```diff
@@ -1,5 +1,5 @@
-import { BaseRepository, getLinkRepository } from "@repositories"
-import { LinkService, getModuleService } from "@services"
+import { BaseRepository, getLinkRepository } from "../repositories"
```

Comment:
question: @carlos-r-l-rodrigues, for some reason, the link module was throwing a `ModuleNotFoundError: Cannot find module '@repositories'` error upon booting the server in my integration tests. Same goes for `@services`.

Do you have any idea why?